satellite/repair/checker/checker.go

```diff
@@ -293,6 +295,14 @@ func (obs *checkerObserver) RemoteSegment(ctx context.Context, path metainfo.Sco
 			}
 		}
 
+		var segmentStart time.Time
+		if pointer.CreationDate.Unix() < pointer.LastRepaired.Unix() {
```

Comment:
Can't you do `if pointer.CreationDate.Before(pointer.LastRepaired)` or something?

Reply:
you're right that's way less ugly